packages/cli/src/utils/settingsUtils.test.ts

```diff
@@ -0,0 +1,636 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { describe, it, expect } from 'vitest';
+import {
+  // Schema utilities
+  getSettingsByCategory,
+  getSettingDefinition,
+  requiresRestart,
+  getDefaultValue,
+  getRestartRequiredSettings,
+  parseSettingKey,
+  getEffectiveValue,
+  getAllSettingKeys,
+  getSettingsByType,
+  getSettingsRequiringRestart,
+  isValidSettingKey,
+  getSettingCategory,
+  // Business logic utilities
+  getSettingValue,
+  isSettingModified,
+  settingExistsInScope,
+  setPendingSettingValue,
+  hasRestartRequiredSettings,
+  getRestartRequiredFromModified,
+  getDisplayValue,
+  isDefaultValue,
+  isValueInherited,
+  getEffectiveDisplayValue,
+} from './settingsUtils.js';
+
+describe('SettingsUtils', () => {
+  // ============================================================================
```

Comment:
nit: make the describe block be 
schema utilities
rather than just having it as a comment.